tests/helics/core/ActionMessage-tests.cpp

```diff
@@ -21,8 +21,9 @@ TEST(ActionMessage_tests, action_test_to_string_conversion)
     BOOST_CHECK_LT(b, 64);
     if (b > 64)
     {
-        printf("sizeof(extraInfo)=%d\n", static_cast<int>(sizeof(std::unique_ptr<ActionMessage::AdditionalInfo>)));
-        printf("payload %d\n", static_cast<int>(reinterpret_cast<char *>(&(m.extraInfo)) - reinterpret_cast<char
+        printf("sizeof(extraInfo)=%d\n",
```

Comment:
Commented code is getting messed up.

Reply:
Yeah I noticed that,   if it ever got uncommented it would probably be pretty simple to fix though